Web Server/conf/htpriv.d/webapp_plugin.htpriv

```diff
@@ -1,2 +1 @@
-ViewResource:users:
-ModifyResource:users:
+ViewResource:everyone:
```

Comment:
Similarly, if we require the use of the `install` command, we don't need this file either.